### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

File: src/infrastructure/commonDiConfig.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ import type { Config } from './config'
 import type { ExternalDependencies } from './diConfig'
 import { SINGLETON_CONFIG } from './diConfig'
 
-export type CommonDiConfig = Record<keyof CommonDependencies, Resolver<unknown>>
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type CommonDiConfig = Record<keyof CommonDependencies, Resolver<any>>
```
Comment (lokankr) [inline src/infrastructure/commonDiConfig.ts:25]: What's the reason for this?

**Review by lokankr — APPROVED** [review]
One question, else looks good 👍

File: src/infrastructure/commonDiConfig.ts (hunk 1 above)
Comment (kibertoad) [reply src/infrastructure/commonDiConfig.ts:25]: https://github.com/kibertoad/awilix-manager/commit/5ca93b6a6fae2d4fc02ff3fd4d22149980490906 this change unfortunately threw off the TS type inference :-/

**Review by dariacm — APPROVED** [review]

File: src/infrastructure/commonDiConfig.ts (hunk 1 above)
Comment (lokankr) [reply src/infrastructure/commonDiConfig.ts:25]: yikes but alright

**Pull request merged**